app/build.gradle

```diff
@@ -22,6 +22,9 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_6
         targetCompatibility JavaVersion.VERSION_1_6
     }
+    lintOptions {
+        abortOnError false
```

Comment:
It's just an example app but I think we shouldn't skip lint checks. What do you think?

Reply:
Totally! actually I got some errors so just adding that 😅

Reply:
I'll address!

Reply:
sorry for the late, addressed! cf4de8c2c9d48d15d6bb634915461a4d0f3037a7